radix-engine-common/src/types/entity_type.rs

```diff
@@ -42,13 +42,16 @@ pub enum EntityType {
     GlobalAccessController = 0b11000011, //---------- 11000 => c, 011xx => vdw0 (011 = access controller)
 
     /// A global native pool entity (196 in decimal). Gives Bech32 prefix: `c` followed by one of `s`, `3`, `j` or `n`.
-    GlobalOneResourcePool = 0b11000100, //---------- 11000 => c, 100xx => s3jn (100 = pool)
+    GlobalOneResourcePool = 0b11000100, //----------- 11000 => c, 100xx => s3jn (100 = pool)
 
     /// A global native pool entity (197 in decimal). Gives Bech32 prefix: `c` followed by one of `5`, `4`, `k` or `h`.
-    GlobalTwoResourcePool = 0b11000101, //---------- 11000 => c, 101xx => 54kh (101 = pool)
+    GlobalTwoResourcePool = 0b11000101, //----------- 11000 => c, 101xx => 54kh (101 = pool)
 
     /// A global native pool entity (197 in decimal). Gives Bech32 prefix: `c` followed by one of `c`, `e`, `6` or `m`.
-    GlobalMultiResourcePool = 0b11000110, //---------- 11000 => c, 110xx => ce6m (101 = pool)
+    GlobalMultiResourcePool = 0b11000110, //--------- 11000 => c, 110xx => ce6m (101 = pool)
+
+    /// A global transaction tracker (130 in decimal). Gives Bech32 prefix: `s` followed by one of `g`, `f`, `2` or `t`.
```

Comment:
Very minor, but in terms of grouping, this should be under `System Components` :)